inductiva/fluids/post_processing/openfoam.py

```diff
@@ -363,12 +374,12 @@ def __init__(self, mesh_data, scalar_name: str = None):
         self.mesh.point_data[scalar_name] = mesh_data.point_data[scalar_name]
         self.mesh.cell_data[scalar_name] = mesh_data.cell_data[scalar_name]
 
-    def render(self,
-               off_screen: bool = False,
-               background_color: str = "black",
-               scalars_cmap: str = "viridis",
-               virtual_display: bool = False,
-               save_path: types.Path = None):
+    def render_frame(self,
```

Comment:
Why renaming this? How is the frame to be rendered selected?

Reply:
I can set render for all. There are no time frames involved.